src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -1433,6 +1435,24 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(
                    is(equalTo("5FCE7321-320B-4EB6-B890-A2AD85B1BFC1")));
     }
 
+    @Test
+    void shouldConvertPartOfAndPartOfSeriesFieldsToPartOfSeriesAndLookUpForSeriesInChannelRegister() {
```

Comment:
This test should probably be a parameterised test with the two variants to make it clear that this is a thing.